scripts/preview_docc.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the Swift Service Discovery open source project
+##
+## Copyright (c) 2022 the SwiftStatsdClient project authors
```

Comment:
ServiceDiscovery? should the soundness script catch this?

Reply:
Yes, there was an issue with the script but it didn't fail:

```
bash... find: Package@swift-5.1.swift: unknown primary or operator
```

Fixed in https://github.com/apple/swift-service-discovery/pull/42/commits/c5e6f560a5da4006ca56c9468f42e3710a66aa5d